web/styles/subscriptions.css

```diff
@@ -116,6 +116,23 @@
     color: hsl(0deg 100% 50%);
 }
 
+.stream_creation_error a,
+#change_stream_name_error a {
+    text-decoration: underline;
+}
+
+#change_stream_name {
+    margin-bottom: 0;
+}
+
+#change_stream_name_error {
+    color: hsl(0deg 100% 50%);
+}
+
+.modal-field-container {
```

Comment:
This is too generic class name. I think something like `change-stream-name-container` would be better.